com.unity.netcode.gameobjects/Runtime/NetworkVariable/Collections/NetworkList.cs

```diff
@@ -39,19 +40,35 @@ public class NetworkList<T> : NetworkVariableBase, IList<T> where T : unmanaged
         /// Creates a NetworkList with a custom value and custom settings
         /// </summary>
         /// <param name="readPerm">The read permission to use for the NetworkList</param>
-        /// <param name="value">The initial value to use for the NetworkList</param>
-        public NetworkList(NetworkVariableReadPermission readPerm, IList<T> value) : base(readPerm)
+        /// <param name="values">The initial value to use for the NetworkList</param>
+        public NetworkList(NetworkVariableReadPermission readPerm, List<T> values) : base(readPerm)
         {
-            m_List = value;
+            foreach (var val in values)
+            {
+                m_List.Add(val);
+            }
         }
 
         /// <summary>
         /// Creates a NetworkList with a custom value and the default settings
         /// </summary>
-        /// <param name="value">The initial value to use for the NetworkList</param>
-        public NetworkList(IList<T> value)
+        /// <param name="values">The initial value to use for the NetworkList</param>
+        public NetworkList(List<T> values)
+        {
+            foreach (var val in values)
+            {
+                m_List.Add(val);
+            }
+        }
+
+        public NetworkList(NetworkVariableReadPermission readPerm, NativeList<T> values) : base(readPerm)
```

Comment:
Matt to check if I can pass in INativeList (or read-only collection,etc.) just for the constructor

Reply:
`IEnumerable<T>` it is :)